### ISSUE TITLE
Implement UI theming

### ISSUE THREAD

**Issue by jamiefeiss** [RDFLib/prez-ui#16]
Theming would allow users to provide custom HTML for the header and footer, as well as custom CSS to customise the look of their Prez UI instance as part of the deployment process.

Custom HTML could be imported outside of the Vue.js app, or by using the v-html element to include HTML in App.vue. Unsure on how doable this would be, and the security issues with importing external HTML. The Vue.js app may require rebuilding depending on the implementation.

For styling, a CSS could be provided for importing, or a simple colour config if we want to restrict styling.

**Comment by edmondchuc** [comment]
See one method of adding custom styling to Prez for Qld Gov Dept of Resources at https://github.com/Kurrawong/qld-resources-prez-ui.

It's probably not the best way to do it but it works for now.

**Comment by jamiefeiss** [comment]
Implemented in 9bb0dc4

**Comment by avillar** [comment]
Are the URLs for the theme templates fixed to "/theme/*"? After deploying a prez-ui instance changing the base url to `/prez-ui/`, I can see that the stylesheets have the prefix (which I guess Vite does automatically), but the header and footer are pulled from `/theme`:

```
[...]
    <link rel="stylesheet" href="[/prez-ui/style.css](view-source:https://defs-stage.opengis.net/prez-ui/style.css)">
    <link rel="stylesheet" href="[/prez-ui/theme/theme.css](view-source:https://defs-stage.opengis.net/prez-ui/theme/theme.css)">
[...]
fetch("/theme/header.html")
            .then(response => {
[...]
```
